kubernetes/ama-logs-geneva.yaml

```diff
@@ -60,6 +60,17 @@ spec:
           valueFrom:
              fieldRef:
                fieldPath: status.hostIP
+       volumeMounts:
+        - mountPath: \etc\config\settings
+          name: settings-vol-config
+          readOnly: true
```

Comment:
Was this needed even with the HPC?

Reply:
Yes. Kubernetes stores config maps in etcd. This mounts it into the node's filesystem